data/conf/mr/hive-site.xml

```diff
@@ -150,6 +150,21 @@
   <value>org.apache.hadoop.hive.ql.hooks.PostExecutePrinter</value>
   <description>Post Execute Hook for Tests</description>
 </property>
+  
+<property>
```

Comment:
i think the whole mr config folder could be dropped

Reply:
There are few sub-tasks pending under https://issues.apache.org/jira/browse/HIVE-28027 
due to which we cant drop the mr folder yet, once those are resolved we can pick that up as separate scope